roles/hiring.md

```diff
@@ -0,0 +1,77 @@
+# Hiring
+
+## Hiring collaboration
+
+[Workable](https://www.workable.com/) is an all-in-one recruiting SaaS. We use it to source and evaluate candidates, track applicants and keep all information in one place so we can then make informed decisions on who to hire.
+
+## Hiring process
+
+### Overview
+
+We strive to have a bureaucracy- and ego-free hiring process that provides a welcoming experience for new Niteans. It's impossible to thoroughly test someone's fit with interviews and phone calls, so we instead do a couple of general tests and then take them on for a [trial period].
+
+That should be your mindset when conducting interviews: you are not searching for an ideal candidate. You are only on the lookout for big "no-no"'s. If someone lacks some skills, they can acquire them. If someone is used to different type of tools or processes, they can learn ours. Look for problem-solving prowess, ability to communicate what went wrong and ownership of mistakes.
+
+### Hiring workflow
+
+Workable guides us through the following stages of hiring a new Nitean. A candidate should expect to wait up to 5 business days between each step of the process.
+
+1. Sourcing
+
+   We post the job ads to various free and paid options, on a per-position basis, but here's a general idea:
+   * Workable posts to 15+ US-based free boards automatically (Glassdoor, Indeed).
+   * We post manually to Slovenian boards: slo-tech.com, internetmojster.com, facebook.com/groups/sluzbezadeveloperje.
+   * We post to twitter.com/teamniteo and facebook.com/niteo.
+   * Sometimes we run an ad campaign on Facebook or buy ads on StackOverflow.
+
+2. Application
+
+   Interested candidates get to our Application Form, hosted by Workable. The form needs to be short, with only general questions. Resist the temptation to ask advanced (technical) questions to asses needed skills for the position. We do these in the next step.
+
+3. Questionnaire
+
+   Members of the hiring team rate (+1/-1) candidates based on application form answers. Whoever gets two or more +1s gets transitioned to the next step: the Questionnaire. Use the `Advanced Questionnaire - <position>` email template to send the questions/tasks to the candidate.
+
+   When the answers arrive we rate the candidate using the scorecard that is prepared in advance and matches the `Advanced Questionnaire - <position>` template. Discussion happens in comments inside Workable, so everything is kept in one place.
+
+   Whoever gets two or more +1s gets transitioned to the next step: Phone Screen.
+
+4. Phone Screen
```

Comment:
Perhaps `Interview Screening`?

Reply:
Interview is the next step, I don't want to cause confusion. Also Workable calls this step phone screen too.